src/Firehose.Web/Authors/AchrafBenAlaya.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class JamesMontemagno : IWorkAtXamarinOrMicrosoft
+    {
+        public string FirstName => "Achraf";
+        public string LastName => "Ben Alaya";
+        public string StateOrRegion => "SFAX, Tunisia";
+        public string EmailAddress => "";
+        public string ShortBioOrTagLine => "Software engineer ,Helping enthusiasts developers to learn mobile (Xamarin Forms & Windows) web (ASP.NET, Web API) ,azure development.";
```

Comment:
Bio might be a bit long. Also see what gets added automatically, see for example the Authors page